sdk/typescript/runtime/tsdistconsts/consts.go

```diff
@@ -0,0 +1,11 @@
+package tsdistconsts
+
+const (
+	DefaultNodeVersion  = "22.11.0" // LTS version, JOD (https://nodejs.org/en/about/previous-releases)
+	nodeImageDigest     = "sha256:b64ced2e7cd0a4816699fe308ce6e8a08ccba463c757c00c14cd372e3d2c763e"
+	DefaultNodeImageRef = "node:" + DefaultNodeVersion + "-alpine@" + nodeImageDigest
+
+	DefaultBunVersion  = "1.1.38"
+	bunImageDigest     = "sha256:5148f6742ac31fac28e6eab391ab1f11f6dfc0c8512c7a3679b374ec470f5982"
+	DefaultBunImageRef = "oven/bun:" + DefaultBunVersion + "-alpine@" + bunImageDigest
+)
```

Comment:
Can't you just put them in a children package of the runtime? Does it make sense to create a go package just for that?

Reply:
i think so. i just started doing it this way before i got answers from you and helder, i'll see about doing it that way.

Reply:
@TomChv im pretty sure this doesn't work bc runtime/ is `package main`?

Reply:
think i got this working here https://github.com/dagger/dagger/pull/9526/commits/4fff7a66617c3551100d8f96f00cfdb8dd3b7717